tests/situations/commun/vues/cadre.test.js

```diff
@@ -154,28 +154,30 @@ describe('Une vue du cadre', function () {
 
   it("enregistre l'événement de fin de situation puis attend la fin de tous les enregistrements", function (done) {
     journal.enregistre = (evenement) => {
-      expect(evenement).to.be.a(EvenementFinSituation);
+      expect(evenement).toBeInstanceOf(EvenementFinSituation);
     };
     journal.attendFinEnregistrement = () => {
       return {
         finally (cb) {
           cb.call();
-          expect(vueCadre.situation.etat()).to.equal(RETOUR_ACCUEIL);
+          expect(vueCadre.situation.etat()).toBe(RETOUR_ACCUEIL);
           done();
         }
       };
     };
     const vueCadre = uneVueCadre();
     vueCadre.situation.emit(CHANGEMENT_ETAT, FINI);
-  });
+  }
+  );
```

Comment:
On peut fusionner ces deux lignes : 
```suggestion
  });
```

Reply:
fait